src/validator.cc

```diff
@@ -1080,35 +1080,35 @@ void Validator::CheckAssertReturnNanCommand(const Action* action) {
 void Validator::CheckCommand(const Command* command) {
```

Comment:
Note: If you made CheckCommand() a (virtual) method of Command, this switch can be removed.